flink/v1.15/flink/src/test/java/com/netease/arctic/flink/read/ArcticSourceTest.java

```diff
@@ -394,7 +394,7 @@ public void testArcticSourceEnumeratorWithChangeExpired() throws Exception {
     LOG.info("commit empty snapshot");
     AppendFiles changeAppend = table.changeTable().newAppend();
     changeAppend.commit();
-    Thread.sleep(ScanContext.MONITOR_INTERVAL.defaultValue().toMillis() * 2);
+    Thread.sleep(MONITOR_INTERVAL.defaultValue().toMillis() * 2);
```

Comment:
Does this unit test also need to use the initArcticSourceWithMonitorInterval function?

Reply:
Good catch.